dart/dynamics/detail/ShapeFrameAspect.hpp

```diff
@@ -88,16 +88,25 @@ struct CollisionAspectProperties
 
 struct DynamicsAspectProperties
 {
-  /// Coefficient of friction
-  double mFrictionCoeff;
+  /// Primary coefficient of friction
+  double mPrimaryFrictionCoeff;
```

Comment:
This breaks backward compatibility since we're changing the name of the public member variable. Could we keep the variable name and add custom functions (i.e., `set/getPrimaryFrictionCoeff`) to `DynamicAspect` instead of `DART_COMMON_SET_GET_ASPECT_PROPERTY(double, PrimaryFrictionCoeff)`?

Reply:
2c506f3145c561d1cc80088029dc10cd7842a79c